clojure-deps/src/main/resources/default-deps.edn

```diff
@@ -129,10 +129,12 @@
  [org.elasticsearch/elasticsearch "5.2.1"]
  [org.elasticsearch.client/transport "5.2.1"]
  [org.hsqldb/hsqldb "2.3.4"]
+ [zookeeper-clj "0.9.4"]
  [org.json/json "20160810"]
  [org.slf4j/slf4j-api "1.7.22"]
  [org.slf4j/slf4j-jdk14 "1.7.22"]
  [org.slf4j/slf4j-log4j12 "1.7.22"]
+ [org.apache.curator/curator-test "2.8.0"]
```

Comment:
@0xbase12 Is this only used for testing?  If so, the scope should be set to "test".

Reply:
It was clear from the other PR that this is only for testing, so I've changed the scope to "test".